src/sentry/api/endpoints/artifact_lookup.py

```diff
@@ -129,8 +116,8 @@ def get(self, request: Request, project: Project) -> Response:
         except SymbolicError:
             pass
         url = request.GET.get("url")
-        release_name = request.GET.get("release")
-        dist_name = request.GET.get("dist")
+        release_name = request.GET.get("release") or ""
+        dist_name = request.GET.get("dist") or ""
```

Comment:
```suggestion
        release_name = request.GET.get("release") or NULL_STRING
        dist_name = request.GET.get("dist") or NULL_STRING
```

I would suggest to use the constant, so that we can hide the sentinel value behind a more idiomatic constant. The constant is in `sentry.models.artifactbundle`.